src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -4152,6 +4159,9 @@
     <Compile Update="$(MSBuildThisFileDirectory)Resources\StaticResource\StaticResource_Simple.xaml.cs">
       <DependentUpon>StaticResource_Simple.xaml</DependentUpon>
     </Compile>
+    <Compile Update="D:\Dev\Uno\src\SamplesApp\UITests.Shared\Windows_UI_StartScreen\JumpListTests.xaml.cs">
```

Comment:
There's a bug in VisualStudio putting absolute paths here.  You need to manually put `$(MSBuildThisFileDirectory)` here.

Reply:
I see, thanks for noticing :-) !